### PR TITLE
[DA] Aligned Danish version of _common.yaml with English version

### PR DESCRIPTION
**Pull request by DocDoo**
Hoping to be able to contribute with Danish translations of sentences, but first time user of Code Spaces so not quite sure if everything went to plan.

### CODE REVIEW

**Comment by fadamsen** [comment]
Hi @DocDoo! This looks good, I’ll take a closer look when I’m at a computer, might not be until tomorrow, though.

**Comment by fadamsen** [comment]
Hi @DocDoo! Life happened, so I'm only looking at this now. I'll post a review in a bit, but note that you'll also need to sign a CLA before this can be merged, as instructed in the comment [here](https://github.com/home-assistant/intents/pull/1252#issuecomment-1474981245).

File: sentences/da/_common.yaml [hunk 1]
```diff
@@ -13,6 +13,12 @@ lists:
       type: percentage
       from: 0
       to: 100
+  brightness_level:
+    values:
+      - in: (max|maksimale|højeste)
+        out: 100
+      - in: (minimum|mindste|laveste)
```
Comment (fadamsen) [inline sentences/da/_common.yaml:20]:
```suggestion
      - in: (lavest[e]|mindst[e]|minimal[e]|minimum)
```

File: sentences/da/_common.yaml [hunk 2]
```diff
@@ -33,6 +39,60 @@ lists:
         out: purple
       - in: brun
         out: brown
+      - in: lyserød
+        out: pink
+      - in: turkis
+        out: turquoise
+  cover_classes:
+    values:
+      - in: markise[rne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:48]:
```suggestion
      - in: markise[(n|r|rne)]
```

File: sentences/da/_common.yaml [hunk 3]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: awning
+      - in: persienne[rne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:50]:
```suggestion
      - in: persienne[(n|r|rne)]
```

File: sentences/da/_common.yaml [hunk 4]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: blind
+      - in: gardin[erne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:52]:
```suggestion
      - in: gardin[(et|er|erne)]
```

File: sentences/da/_common.yaml [hunk 5]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: curtain
+      - in: dør[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:54]:
```suggestion
      - in: dør[(en|e|ene)]
```

File: sentences/da/_common.yaml [hunk 6]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: door
+      - in: garageport[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:56]:
```suggestion
      - in: garage(dør|port)[(en|e|ene)]
```

File: sentences/da/_common.yaml [hunk 7]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: garage
+      - in: låge[rne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:58]:
```suggestion
      - in: låge[(en|r|rne)]
```

File: sentences/da/_common.yaml [hunk 8]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: gate
+      - in: skærm[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:60]: This is closer to "persienne" or "rullegardin" in Danish.

```suggestion
      - in: (persienne[(en|r|rne)]|rullegardin[(et|er|erne)])
```

File: sentences/da/_common.yaml [hunk 9]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: shade
+      - in: skodde[rne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:62]:
```suggestion
      - in: skodde[(n|r|rne)]
```

File: sentences/da/_common.yaml [hunk 10]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: shutter
+      - in: vindue[rne]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:64]:
```suggestion
      - in: vindue[(t|r|rne)]
```

File: sentences/da/_common.yaml [hunk 11]
```diff
@@ -33,6 +39,60 @@ lists:
...
+    values:
+      - in: "tændt"
+        out: "on"
+      - in: "slukket"
+        out: "off"
+  on_off_domains:
+    values:
+      - in: lys[ene]
+        out: light
+      - in: ventilator[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:92]:
```suggestion
      - in: ventilator([en|erne])
```

File: sentences/da/_common.yaml [hunk 12]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: fan
+      - in: afbryder[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:94]:
```suggestion
      - in: (afbryder[(en|ne)]|kontakt[(en|erne)])
```

File: sentences/da/_common.yaml [hunk 13]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_battery_charging_states:
+    values:
+      - in: "lader"
+        out: "on"
+      - in: "lader ikke"
+        out: "off"
+  bs_carbon_monoxide_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:125]: I don't think "påvist" and "tilstedeværende" make sense, semantically, for this.

```suggestion
      - in: "(detekteret|fundet|registreret)"
```

File: sentences/da/_common.yaml [hunk 14]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:127]:
```suggestion
      - in: "(ikke detekteret|ikke fundet|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 15]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_cold_states:
+    values:
+      - in: "koldt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:131]:
```suggestion
      - in: "kold[t]"
```

File: sentences/da/_common.yaml [hunk 16]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_garage_door_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_gas_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:155]:
```suggestion
      - in: "(detekteret|fundet|registreret)"
```

File: sentences/da/_common.yaml [hunk 17]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:157]:
```suggestion
      - in: "(ikke detekteret|ikke fundet|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 18]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_heat_states:
+    values:
+      - in: "varmt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:161]:
```suggestion
      - in: "varm[t]"
```

File: sentences/da/_common.yaml [hunk 19]
```diff
@@ -46,10 +106,176 @@ lists:
...
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_cold_states:
+    values:
+      - in: "koldt"
+        out: "on"
+      - in: "normalt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:133]:
```suggestion
      - in: "normal[t]"
```

File: sentences/da/_common.yaml [hunk 20]
```diff
@@ -46,10 +106,176 @@ lists:
...
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_heat_states:
+    values:
+      - in: "varmt"
+        out: "on"
+      - in: "normalt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:163]:
```suggestion
      - in: "normal[t]"
```

File: sentences/da/_common.yaml [hunk 21]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_light_states:
+    values:
+      - in: "detekteret"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:167]:
```suggestion
      - in: "(detekteret|registreret)"
```

File: sentences/da/_common.yaml [hunk 22]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "ikke detekteret"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:169]:
```suggestion
      - in: "(ikke detekteret|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 23]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_lock_states:
+    values:
+      - in: "ulåst"
+        out: "on"
+      - in: "låst"
+        out: "off"
+  bs_moisture_states:
+    values:
+      - in: "vådt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:179]:
```suggestion
      - in: "våd[t]"
```

File: sentences/da/_common.yaml [hunk 24]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "tørt"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:181]:
```suggestion
      - in: "tør[t]"
```

File: sentences/da/_common.yaml [hunk 25]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_motion_states:
+    values:
+      - in: "(detekteret|påvist)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:185]:
```suggestion
      - in: "(detekteret|registreret)"
```

**Comment by DocDoo** [comment]
> Hi @DocDoo! Life happened, so I'm only looking at this now. I'll post a review in a bit, but note that you'll also need to sign a CLA before this can be merged, as instructed in the comment [here](https://github.com/home-assistant/intents/pull/1252#issuecomment-1474981245).

👍, no problem, Life does indeed happen🙂
Regarding the CLA, the cla-bot reached out and we got the agreement signed by me, so it looks (to me) like this is already in order, but let me know if this is not the case.
image

**Comment by fadamsen** [comment]
Ah, I missed that! I thought there would be a comment, but all is well.

File: sentences/da/_common.yaml [hunk 26]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:187]:
```suggestion
      - in: "(ikke detekteret|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 27]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_occupancy_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:191]:
```suggestion
      - in: "(detekteret|registreret)"
```

File: sentences/da/_common.yaml [hunk 28]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:193]:
```suggestion
      - in: "(ikke detekteret|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 29]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_opening_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_plug_states:
+    values:
+      - in: "tilkoblet"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:203]:
```suggestion
      - in: "(koblet til|sat i)"
```

File: sentences/da/_common.yaml [hunk 30]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "frakoblet"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:205]:
```suggestion
      - in: "(ikke koblet til|ikke sat i)"
```

File: sentences/da/_common.yaml [hunk 31]
```diff
@@ -46,10 +106,176 @@ lists:
...
+    values:
+      - in: "tændt"
+        out: "on"
+      - in: "slukket"
+        out: "off"
+  bs_presence_states:
+    values:
+      - in: "(hjemme|tilstedeværende)"
+        out: "on"
+      - in: "(væk|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:217]:
```suggestion
      - in: "(fraværende|ikke hjemme|ude)"
```

File: sentences/da/_common.yaml [hunk 32]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_problem_states:
+    values:
+      - in: "fundet"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:221]:
```suggestion
      - in: "(detekteret|fundet|registreret)"
```

File: sentences/da/_common.yaml [hunk 33]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "intet fundet"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:223]:
```suggestion
      - in: "(ikke detekteret|ikke fundet|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 34]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_running_states:
+    values:
+      - in: "(kører|aktiveret)"
+        out: "on"
+      - in: "(stoppet|deaktiveret)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:229]:
```suggestion
      - in: "kører ikke"
```

File: sentences/da/_common.yaml [hunk 35]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_problem_states:
+    values:
+      - in: "fundet"
+        out: "on"
+      - in: "intet fundet"
+        out: "off"
+  bs_running_states:
+    values:
+      - in: "(kører|aktiveret)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:227]: I don't think "aktiveret" makes sense for this.

```suggestion
      - in: "kører"
```

File: sentences/da/_common.yaml [hunk 36]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(stoppet|deaktiveret)"
+        out: "off"
+  bs_safety_states:
+    values:
+      - in: "farlig"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:233]:
```suggestion
      - in: "(farlig|usikker)"
```

File: sentences/da/_common.yaml [hunk 37]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "sikker"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:235]:
```suggestion
      - in: "(sikker|ufarlig)"
```

File: sentences/da/_common.yaml [hunk 38]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_smoke_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:239]:
```suggestion
      - in: "(detekteret|fundet|registreret)"
```

File: sentences/da/_common.yaml [hunk 39]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:241]:
```suggestion
      - in: "(ikke detekteret|ikke fundet|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 40]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_sound_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:245]:
```suggestion
      - in: "(detekteret|registreret)"
```

File: sentences/da/_common.yaml [hunk 41]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:247]:
```suggestion
      - in: "(ikke detekteret|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 42]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_tamper_states:
+    values:
+      - in: "(detekteret|pillet ved)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:251]:
```suggestion
      - in: "(detekteret|pillet ved|registreret)"
```

File: sentences/da/_common.yaml [hunk 43]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|urørt)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:253]:
```suggestion
      - in: "(ikke detekteret|ikke pillet ved|ikke registreret)"
```

File: sentences/da/_common.yaml [hunk 44]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_update_states:
+    values:
+      - in: "opdatering tilgængelig"
+        out: "on"
+      - in: "opdateret"
+        out: "off"
+  bs_vibration_states:
+    values:
+      - in: "(detekteret|vibrerer)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:263]:
```suggestion
      - in: "(detekteret|registreret|vibrerer)"
```

File: sentences/da/_common.yaml [hunk 45]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "(ikke detekteret|vibrerer ikke)"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:265]:
```suggestion
      - in: "(ikke detekteret|ikke registreret|vibrerer ikke)"
```

File: sentences/da/_common.yaml [hunk 46]
```diff
@@ -46,10 +106,176 @@ lists:
...
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "closed"
+
 expansion_rules:
   al: "(alle | alt)"
   blæser: "blæser[(e|en|ne)]"
   garagedør: "garagedør[(en|e|ene)]"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:277]: Both "garagedør" and "garageport" represent the same device class, so I don't think they need to be treated as separate expansion rules.

```suggestion
  garagedør: "garage(dør|port)[(en|e|ene)]"
```

File: sentences/da/_common.yaml [hunk 47]
```diff
@@ -46,10 +106,176 @@ lists:
...
+  garageport: "garageport[(en|e|ene)]"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:278]: Combined with "garagedør" above.

```suggestion
```

File: sentences/da/_common.yaml [hunk 48]
```diff
@@ -65,4 +291,7 @@ expansion_rules:
   temperatur: "{temperature}[°| grader] [{temperature_unit}]"
   tænd: "tænd [for]"
   åbn: "åbn [for]"
-skip_words: []
+
+skip_words:
+  - "vær så venlig"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:296]:
```suggestion
  - "venligst"
```

File: sentences/da/_common.yaml [hunk 49]
```diff
@@ -65,4 +291,7 @@ expansion_rules:
...
+  - "vil du"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:297]:
```suggestion
  - "vil du"
  - "vær så venlig"
```

File: sentences/da/_common.yaml [hunk 50]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: gate
+      - in: skærm[ene]
+        out: shade
+      - in: skodde[rne]
+        out: shutter
+      - in: vindue[rne]
+        out: window
+  cover_states:
+    values:
+      - in: "åben"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:68]:
```suggestion
      - in: "(åben[t]|åbne[t])"
```

File: sentences/da/_common.yaml [hunk 51]
```diff
@@ -33,6 +39,60 @@ lists:
...
+        out: "unlocked"
+  on_off_states:
+    values:
+      - in: "tændt"
+        out: "on"
+      - in: "slukket"
+        out: "off"
+  on_off_domains:
+    values:
+      - in: lys[ene]
```
Comment (fadamsen) [inline sentences/da/_common.yaml:90]:
```suggestion
      - in: lys[(et|ene)]
```

File: sentences/da/_common.yaml [hunk 52]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_vibration_states:
+    values:
+      - in: "(detekteret|vibrerer)"
+        out: "on"
+      - in: "(ikke detekteret|vibrerer ikke)"
+        out: "off"
+  bs_window_states:
+    values:
+      - in: "åben"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:269]:
```suggestion
      - in: "(åben[t]|åbne[t])"
```

File: sentences/da/_common.yaml [hunk 53]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_connectivity_states:
+    values:
+      - in: "forbundet"
+        out: "on"
+      - in: "afbrudt"
+        out: "off"
+  bs_door_states:
+    values:
+      - in: "åben"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:143]:
```suggestion
      - in: "(åben[t]|åbne[t])"
```

File: sentences/da/_common.yaml [hunk 54]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_garage_door_states:
+    values:
+      - in: "åben"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:149]:
```suggestion
      - in: "(åben[t]|åbne[t])"
```

File: sentences/da/_common.yaml [hunk 55]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_light_states:
+    values:
+      - in: "detekteret"
+        out: "on"
+      - in: "ikke detekteret"
+        out: "off"
+  bs_lock_states:
+    values:
+      - in: "ulåst"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:173]:
```suggestion
      - in: "(låst op|ulåst)"
```

File: sentences/da/_common.yaml [hunk 56]
```diff
@@ -46,10 +106,176 @@ lists:
...
+        out: "off"
+  bs_occupancy_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_opening_states:
+    values:
+      - in: "åben"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:197]:
```suggestion
      - in: "(åben[t]|åbne[t])"
```

**Review by fadamsen — CHANGES_REQUESTED** [review]
Okay, quite a few comments here, but mostly small stuff, such as adding more alternatives and word endings to align with the existing translations.

A couple of words are, I think, not very natural to use, especially "påvist" and "tilstedeværende" are used in many places where, to me, they don't make sense. It's hard to draw the line between being accommodating and making things to complicated, but I tried.

Do let me know if there is anything in particular you disagree with and we can discuss.

**Comment by fadamsen** [comment]
All right, I'm merging my suggestions so we can get this closed.

File: sentences/da/_common.yaml [hunk 57]
```diff
@@ -65,4 +290,9 @@ expansion_rules:
   temperatur: "{temperature}[°| grader] [{temperature_unit}]"
   tænd: "tænd [for]"
   åbn: "åbn [for]"
-skip_words: []
+
+skip_words:
+  - "venligst"
+  - "vil du"
+  - "vær så venlig"
+  - "vær så venlig"
```
Comment (fadamsen) [inline sentences/da/_common.yaml:298]:
```suggestion
```

**Review by fadamsen — APPROVED** [review]

**Comment by fadamsen** [comment]
Thank you for the help, @DocDoo!

**Comment by DocDoo** [comment]
> All right, I'm merging my suggestions so we can get this closed.

Yeah, sorry about not getting back to you @fadamsen , but now I got caught up in lot of other personal stuff. Thanks for pushing this through and I will take a look at your comments and suggestions which are very reasonable and come with any further comments. I'm also trying to get started in providing Danish translations of the sentences for "Lock", but struggling a bit as I also need to create the relevant files in the folder for the Danish language files. I think it has to do with me not entirely understanding how tests fit together with sentences, but I'll think some more about it and will possibly ask you for help if I don't end up figuring it out myself :-)

**Comment by fadamsen** [comment]
No worries, I know how it is!

**Pull request merged**